scalardl/src/scalardl/core.clj

```diff
@@ -66,7 +66,7 @@
                           (if-let [config (some-> test
                                                   create-client-properties
                                                   ClientConfig.)]
-                            (doto (ClientServiceFactory.) (.create config))
+                            (-> (ClientServiceFactory.) (.create config))
```

Comment:
I wanted to return a ClientSerice object but it was returning ClientServiceFactory object.
Sorry, it was my Clojure issue.